lighthouse-core/lib/sentry.js

```diff
@@ -74,6 +76,19 @@ function init(opts) {
       // @ts-ignore Non-standard property added to flag error as not needing capturing.
       if (err.expected) return;
 
+      const tags = opts.tags || {};
+      if (tags.audit) {
+        const key = `audit-${tags.audit}-${err.message}`;
+        if (sentryExceptionCache.has(key)) return;
```

Comment:
Just curious:  Could 1 exception have both audit and gatherer tags?  In which case returning here would not allow the gatherer tags to be analyzed?  I don't think so, but wanted to make sure.

Reply:
nope, it either happens in an audit, or a gatherer, or neither :)